src/panels/lovelace/badges/hui-entity-filter-badge.ts

```diff
@@ -109,11 +109,14 @@ class EntityFilterBadge extends HTMLElement implements LovelaceBadge {
       return;
     }
 
-    // Attach element if it has never been attached.
-    if (!this.lastChild) {
-      for (const element of this._elements) {
-        this.appendChild(element);
-      }
+    let child = this.lastElementChild;
```

Comment:
Can we use `lastChild`? Browser support is better and I don't think we need `lastElementChild`?